src/gt4py/cartesian/frontend/gtscript_frontend.py

```diff
@@ -692,6 +692,18 @@ class ParsingContext(enum.Enum):
     COMPUTATION = 2
 
 
+def _is_absolute_indexing_name(name: str):
```

Comment:
maybe
```suggestion
def _is_datadim_indexing_name(name: str):
```
because otherwise it might be understood as absolute indexing in i,j,k

Reply:
Renamed the code to follow the `datadims` to be cleaner. Also renamed PR to showcase the "data dims direct access" which is a big part of the feature